plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/IcebergRESTCatalogModule.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.rest;
+
+import com.google.inject.Binder;
+import com.google.inject.Scopes;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.trino.plugin.iceberg.catalog.TrinoCatalogFactory;
+
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static org.weakref.jmx.guice.ExportBinder.newExporter;
+
+public class IcebergRESTCatalogModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    protected void setup(Binder binder)
+    {
+        configBinder(binder).bindConfig(RESTIcebergConfig.class);
+        binder.bind(TrinoCatalogFactory.class).to(TrinoIcebergRESTCatalogFactory.class).in(Scopes.SINGLETON);
+        newExporter(binder).export(TrinoCatalogFactory.class).withGeneratedName();
```

Comment:
You can remove this line, it's for exporting JMX stats, but it doesn't look like TrinoIcebergRESTCatalogFactory has any
```suggestion
```